infra/src/key_vault.tf

```diff
@@ -33,6 +33,11 @@ data "azurerm_key_vault_secret" "serviceid_quality_check_exclusion_list" {
   key_vault_id = module.key_vault_domain.id
 }
 
+data "azurerm_key_vault_secret" "userid_cms_to_legacy_sync_inclusion_list" {
+  name         = "USERID-CMS-TO-LEGACY-SYNC-INCLUSION-LIST"
+  key_vault_id = module.key_vault_domain.id
+}
+
```

Comment:
why a list of user id is a secret? I mean, I'm pretty sure they are not sensitive datas, so there is no need to put them in the key vault

Reply:
bc9c699